src/constants/actions.js

```diff
@@ -1,11 +1,11 @@
 export const MOVIES = {
-  SEARCH: 'SEARCH_MOVIES',
   SEARCH_RESET: 'SEARCH_MOVIES_RESET',
   LOAD_SUCCESS: 'LOAD_MOVIES_SUCCESS',
   TITLE: 'GET_PAGE_TITLE',
-  SEARCH_QUERY: 'GET_SEARCH_QUERY',
+  INIT_SEARCH: 'INIT_SEARCH_PROPS',
   REQUEST: 'LOAD_MOVIES_REQUEST',
   FAILURE: 'LOAD_MOVIES_FAILURE',
+  SET_PAGE: 'SET_PAGINATION',
```

Comment:
`'SET_PAGE'` instead `'SET_PAGINATION'` ? You can name variables the same as their values, it's not a problem :)